swagger_to_sdk_config.json

```diff
@@ -6,7 +6,7 @@
   },
   "projects": {
     "azure_mgmt_analysis_services": {
-      "swagger": "arm-analysisservices/2016-05-16/swagger/analysisservices.json",
+      "swagger": "specification/analysisservices/resource-manager/readme.md",
```

Comment:
should we be passing a tag instead of relying on the default in the readme file. I think that would help us be explicit on what group of files we're actually picking up for generation. If not we'd have to relay purely on commit dates and changes history on the readme file to know what we generated from last time. 
Is there any drawback on actually passing a tag explicitly from our side?